src/EFCore.PG/Scaffolding/Internal/DbDataReaderExtension.cs

```diff
@@ -1,27 +1,35 @@
 ﻿using System.Data.Common;
 using System.Diagnostics;
-using JetBrains.Annotations;
+using System.Diagnostics.CodeAnalysis;
+
+#nullable enable
 
 namespace Npgsql.EntityFrameworkCore.PostgreSQL.Scaffolding.Internal
 {
-    public static class DbDataReaderExtension
+    static class DbDataReaderExtension
     {
         [DebuggerStepThrough]
-        public static T GetValueOrDefault<T>([NotNull] this DbDataReader reader, [NotNull] string name)
+        [return: MaybeNull]
+        internal static T GetValueOrDefault<T>(this DbDataReader reader, string name)
```

Comment:
In which case a reader is not `NpgsqlDataReader`?

Reply:
In our code I think it's always NpgsqlDataReader - but it doesn't really make a difference.. Any reason to chance it?

Reply:
Since `NpgsqlDataReader` is a final class the JIT could inline calls if they meet requirements. Another reason is that it's an internal extension method which is for Npgsql only, no need in an additional abstraction.